lightning_pose/data/utils.py

```diff
@@ -395,18 +395,11 @@ def generate_heatmaps(
     confidence *= -1
     confidence /= 2 * sigma**2
     confidence = torch.exp(confidence)
```

Comment:
@colehurwitz what is `confidence` at this stage for a NaN keypoint? just making sure we have no surprises

Reply:
i was thinking about this last time I looked at this function - we should just rename this variable `heatmap` since that's what it actually is

Reply:
Changed the name to heatmap. Honestly, I have no idea what confidence/heatmaps are for NaN points before zeroing.